.yarnrc.yml

```diff
@@ -2,4 +2,4 @@ nodeLinker: node-modules
 
 npmPublishAccess: public
 
-yarnPath: .yarn/releases/yarn-4.0.2.cjs
+yarnPath: .yarn/releases/yarn-4.5.3-git.20241125.hash-8bfe2d5.cjs
```

Comment:
Yarn bump intentional?

Reply:
yeah, cross-spawn is a deep dep of yarn that was pinned to an old version, so I had to upgrade yarn on most repos